app/jobs/notify_partner_job.rb

```diff
@@ -0,0 +1,7 @@
+class NotifyPartnerJob < ApplicationJob
+  def perform(request_id)
+    request = Request.find(request_id)
+
+    RequestsConfirmationMailer.confirmation_email(request).deliver_now
```

Comment:
Same here. Should be deliver_later

Reply:
I agree, should be `deliver_later` I think perform_now in the prior example should be okay?